src/main/java/org/checkerframework/languageserver/CFLanguageServer.java

```diff
@@ -17,19 +17,28 @@
 
 /** The actual language server, responsible for communicating with the client (editor). */
 public class CFLanguageServer implements LanguageServer, LanguageClientAware {
-
+    /** The logger for issuing information in the language server. */
     private static final Logger logger = Logger.getLogger(CFLanguageServer.class.getName());
 
-    // used for identifying the source of diagnostics, or field for settings
+    /** The String used for identifying the source of diagnostics, or field for settings. */
```

Comment:
What does `field for settings` mean? Is it the key in the settings file?

Reply:
I think it should be removed as it is not used anywhere else.

Reply:
https://github.com/search?q=repo%3Aeisopux%2Fchecker-framework-languageserver%20SERVER_NAME&type=code shows two uses of the field.

Reply:
yes, I think this is also used for indentifying the source of the diagnostic as  it use the constant field for both serialization and deserialization of the object.
```
@SerializedName(CFLanguageServer.SERVER_NAME)
    final Config config;
```